apps/examples/examples.coffee

```diff
@@ -3,158 +3,105 @@ module.exports = [
     id: 'mood'
     title: 'Mood'
     description: 'Vibes, patterns, atmospheres, colors, and more'
-    # channel_ids: [
-    #   "sublime",
-    #   "color-schemes",
-    #   "hotel-concepts",
-    #   "color-files",
-    #   "life-luxury",
-    # ]
-    channel_ids: [13771, 28613, 7469, 158542, 46371]
+    # channel_ids: ["year-zero", "color-files", "color-schemes", "hotel-concepts", "life-luxury"]
+    channel_ids: [37968, 28613, 7469, 158542, 46371]
```

Comment:
A simple thing but if you want to maintain a mapping between name and ID it'd probably alot clearer to use an object as a map:
```javascript
IDS = {
  "year-zero": 37968,
  "color-files": 28613,
  "color-schemes": 7469,
  "hotel-concepts": 158542,
  // ...
}

channel_ids: [
  IDS["year-zero"],
  IDS["color-files"],
  IDS["color-schemes"],
  IDS["hotel-concepts"],
  IDS["life-luxury"]
]
```

Reply:
Ah yeah, good call.